meetings/2020/WASI-02-27.md

```diff
@@ -0,0 +1,37 @@
+![WASI logo](/WASI.png)
+
+## Agenda for the February 27 video call of WASI Subgroup
+
+- **Where**: zoom.us
+- **When**: February 27, 17:00-18:00 UTC
+- **Location**: *link on calendar invite*
+- **Contact**:
+    - Name: Dan Gohman
+    - Email: sunfish@mozilla.com
+
+### Registration
+
+None required if you've attended before. Email Dan Gohman to sign up if it's
+your first time. The meeting is open to CG members only.
+
+## Logistics
+
+The meeting will be on a zoom.us video conference.
+Installation is required, see the calendar invite.
+
+## Agenda items
+
+1. Opening, welcome and roll call
+    1. Please help add your name to the meeting notes.
+    1. Please help take notes.
+    1. Thanks!
+1. Proposals and discussions
+    1. Whole-program LTO coming:
+        - https://github.com/WebAssembly/tool-conventions/pull/134
```

Comment:
Is this the right URL? this hasnt been updated in a year and seems to be talking about ewasm